wewatch-client/src/components/Join/Join.js

```diff
@@ -1,24 +1,24 @@
 import React, { useState } from 'react';
 import { Link } from "react-router-dom";
+import queryString from 'query-string';
 
 import './Join.css';
 
 export default function SignIn() {
-  const [name, setName] = useState('');
+  
+
   const [room, setRoom] = useState('');
+  const { name } = queryString.parse(window.location.search);
```

Comment:
I think that location.search was called directly in a similar use case in Chat.js, would it be possible to maintain that structure here too? I think the location object was provided as a parameter to the exported function in the case of Chat.js